app/models/comment.rb

```diff
@@ -1,4 +1,4 @@
 class Comment < ApplicationRecord
-  belongs_to :author_id
-  belongs_to :post_id
+  belongs_to :author, class_name: :User, foreign_key: :author_id
+  belongs_to :post, foreign_key: :post_id
```

Comment:
[REQUIRED] Kindly include a method that updates the comment counter for a post because this is a requirement